src/librustc_privacy/lib.rs

```diff
@@ -782,6 +782,10 @@ impl<'a, 'tcx> Visitor<'tcx> for NamePrivacyVisitor<'a, 'tcx> {
         NestedVisitorMap::All(&self.tcx.hir())
     }
 
+    fn visit_mod(&mut self, _m: &'tcx hir::Mod, _s: Span, _n: ast::NodeId) {
+        // Don't visit modules inside
```

Comment:
Can explain in the comment why we are not recursing down into nested modules? Something like "Don't visit nested modules, since we run a separate visitor walk for each module in `privacy_access_levels()`".